ts-client/src/constants.ts

```diff
@@ -10,3 +10,8 @@ export const ERROR = {
   POOL_NOT_LOAD: "Pool not loaded",
   INVALID_MINT: "Invalid mint",
 };
+
+export const EXTRA_ACCOUNTS = {
+  marinade: [new PublicKey("8szGkuLTAux9XMgZ2vtY39jVSowEcpBfFfD8hXSEqdGC")],
```

Comment:
Could you comment how the account is used for

Reply:
Added comment